mlflow/server/js/src/experiment-tracking/components/CompareRunView.js

```diff
@@ -37,6 +37,40 @@ export class CompareRunView extends Component {
     intl: PropTypes.shape({ formatMessage: PropTypes.func.isRequired }).isRequired,
   };
 
+  constructor(props) {
+    super(props);
+    this.state = {
+      tableContainerWidth: null,
+      collapseParamBlock: false,
+      collapseMetricBlock: false,
```

Comment:
```suggestion
```

We probably don't need these two states once https://github.com/mlflow/mlflow/pull/5306/files#r795070842 is addressed.

Reply:
+1